[2024-04-20T15:29:45.262+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-20T15:29:45.277+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2024-01-20T00:00:00+00:00 [queued]>
[2024-04-20T15:29:45.281+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2024-01-20T00:00:00+00:00 [queued]>
[2024-04-20T15:29:45.281+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-20T15:29:45.287+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): first_task> on 2024-01-20 00:00:00+00:00
[2024-04-20T15:29:45.291+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1151) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-20T15:29:45.291+0000] {standard_task_runner.py:63} INFO - Started process 1152 to run task
[2024-04-20T15:29:45.291+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2024-01-20T00:00:00+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp2nnp8dfc']
[2024-04-20T15:29:45.292+0000] {standard_task_runner.py:91} INFO - Job 177: Subtask first_task
[2024-04-20T15:29:45.315+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2024-01-20T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-20T15:29:45.380+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-20T00:00:00+00:00'
[2024-04-20T15:29:45.381+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-20T15:29:45.394+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-20T15:29:45.394+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-04-20T15:29:45.399+0000] {subprocess.py:86} INFO - Output:
[2024-04-20T15:29:45.401+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-04-20T15:29:45.401+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-20T15:29:45.401+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-20T15:29:45.413+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, execution_date=20240120T000000, start_date=20240420T152945, end_date=20240420T152945
[2024-04-20T15:29:45.436+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-20T15:29:45.445+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-20T15:29:45.446+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
